test/Bouncer.test.js

```diff
@@ -0,0 +1,67 @@
+
+import assertRevert from './helpers/assertRevert';
+
+const Bouncer = artifacts.require('BouncerMock');
+
+require('chai')
+  .use(require('chai-as-promised'))
+  .should();
+
+const getSigner = (contract, signer) => (addr) => {
+  // via: https://github.com/OpenZeppelin/zeppelin-solidity/pull/812/files
+  const message = contract.address.substr(2) + addr.substr(2);
+  // ^ substr because in solidity the address is a set of byes, not a string `0xabcd`
+  return web3.eth.sign(signer, web3.sha3(message, { encoding: 'hex' }));
+};
+
+contract('Bouncer', ([owner, authorizedUser, anyone, bouncerAddress]) => {
```

Comment:
I generally like assigning `accounts[0]` to a `_` variable, because it's the address used by truffle-contract by default. In this case, so as not to accidentally send a tx from the owner unless explicitly specified.